src/components/NewGalaxyModal.js

```diff
@@ -203,8 +203,8 @@ export const NewGalaxyModal = _.flow(
   }
 
   const renderDefaultCase = () => {
-    const { cpu, memory } = _.find({ name: 'n1-standard-8' }, machineTypes)
-    const cost = getGalaxyCost(app || { kubernetesRuntimeConfig: { machineType: 'n1-standard-8', numNodes: 2 } })
+    const { cpu, memory } = _.find({ name: 'n1-standard-8' }, machineBases)
+    const cost = getGalaxyCost(app || { status: 'RUNNING', kubernetesRuntimeConfig: { machineType: 'n1-standard-8', numNodes: 2 } })
```

Comment:
It feels weird to me to have a non-existant app have a pseudo  status of running. I know it's only for the cost calculation but am wondering if since it's only within this context would it be less "hacky" to use a fake status of "WILL_RUN" or something similar.. or is that more "hacky" ?

Reply:
Yea, I had the same feeling when I added it. Using a fake status may indeed be less misleading. I think I'll go with that unless I can find a better solution.

Reply:
Feels like `UNKNOWN` may represent the situation a bit better and would not require introducing a fake status. If any thoughts, let me know.

Reply:
Actually looking at this more I think the most standard way to deal with this would be to utilize `undefined` which would require a check for and `app` and an `app.status` when setting `appStatus` and then applying the uppercase to that ie:
`app && app.status ? app.status.toUpperCase() : undefined` 

then below you could just use `switch (appStatus)` and you also wouldn't need to add a status to the 'fake' app in runtime-utils

Reply:
`app?.status?.toUpperCase()` will default to `undefined` if any part of it is, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining